packages/react/src/codemods/styled-components-to-compiled/__tests__/styled-components-to-compiled.test.tsx

```diff
@@ -79,6 +79,25 @@ describe('styled-components-to-compiled transformer', () => {
     'it should not remove comments before transformed statement when not on top'
   );
 
+  defineInlineTest(
+    { default: transformer, parser: 'tsx' },
+    {},
+    `
+    import styled, { css, keyframes, createGlobalStyle, ThemeProvider, withTheme } from 'styled-components';
+    import * as React from 'react';
+    `,
+    `
+    /* TODO(@compiled/react codemod): "css" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    /* TODO(@compiled/react codemod): "keyframes" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    /* TODO(@compiled/react codemod): "createGlobalStyle" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
```

Comment:
only the things that will break usage should be commented i think

so `css` and `keyframes` - everything else wouldn't break style usage so we could remove them for now.

Reply:
🤔 What about other api's which are exported from `styled-components`? They will definitely break the code right? These comments ensure that codemod is smart enough to figure out all the exports which are now unresolved since we have converted import path from `styled-component` to `@compiled/react`. And it will be easier for user to figure out api's which are removed from `styled-components`.

Reply:
hmmm true okay 😄